src/main/kotlin/ua/marchenko/artauction/testComponent/TestController.kt

```diff
@@ -1,4 +1,4 @@
-package ua.marchenko.artauction.common.annotation
+package ua.marchenko.artauction.testComponent
```

Comment:
I meant into "src/test/...":)

Reply:
I have tried to put it in test, but classes in this folder doesn't work like general rest api component.
This classes are made to show the usage of custom annotations, so I want them to be access from postman etc